pkg/validate/security_context.go

```diff
@@ -428,3 +464,38 @@ func compareNetworkList(podConfig *runtimeapi.PodSandboxConfig, logPath string,
 	}
 
 }
+
+// createPrivilegedContainer creates container with specified Privileged in ContainerConfig.
+func createPrivilegedContainer(rc internalapi.RuntimeService, ic internalapi.ImageManagerService, podID string, podConfig *runtimeapi.PodSandboxConfig, prefix string, privileged bool) (string, string) {
+	containerName := prefix + framework.NewUUID()
+	containerConfig := &runtimeapi.ContainerConfig{
+		Metadata: buildContainerMetadata(containerName, defaultAttempt),
+		Image:    &runtimeapi.ImageSpec{Image: defaultContainerImage},
+		Command:  []string{"top"},
+		Linux: &runtimeapi.LinuxContainerConfig{
+			SecurityContext: &runtimeapi.LinuxContainerSecurityContext{
+				Privileged: privileged,
+			},
+		},
+	}
+
+	return createContainer(rc, ic, containerConfig, podID, podConfig), containerName
+}
+
+// verifyPrivileged verifies the container with Privileged config works fine.
+func verifyPrivileged(rc internalapi.RuntimeService, containerID string, isPrivileged bool) {
+	cmd := []string{"ip", "link", "add", "dummy0", "type", "dummy"}
+	reverseCmd := []string{"ip", "link", "del", "dummy0"}
+
+	stdout, stderr, err := rc.ExecSync(containerID, cmd, time.Duration(defaultExecSyncTimeout)*time.Second)
+	msg := fmt.Sprintf("cmd %v, stdout %q, stderr %q", cmd, stdout, stderr)
+
+	if isPrivileged {
+		Expect(err).NotTo(HaveOccurred(), msg)
+
+		// We need to clean up the dummy link that was created.
+		execSyncContainer(rc, containerID, reverseCmd)
```

Comment:
I think this is not required since we will remove the container/sandbox after each test case.